src/components/Output.tsx

```diff
@@ -368,7 +369,7 @@ export const Output = ({
   }
 };
 
-const _OUTPUT_STYLES = styled.output<{ withParentheses?: boolean }>`
+const OUTPUT_STYLES = styled.output<{ withParentheses?: boolean }>`
```

Comment:
since we've transitioned to individual style declarations, we can probably just throw this into the `const $Text` assignment and have `$Number` extend `$Text`